sdk/appconfiguration/azure-appconfiguration/tests/test_azure_configuration_client_async.py

```diff
@@ -25,26 +25,81 @@
     PAGE_SIZE,
     KEY_UUID,
 )
-from async_proxy import AzureAppConfigurationClientProxy
 import pytest
 import datetime
+import os
+import logging
 
-class AppConfigurationClientTest(AzureAppConfigurationClientTestBase):
+class AppConfigurationClientTest(AzureMgmtTestCase):
     def __init__(self, method_name):
-        super(AppConfigurationClientTest, self).__init__(
-            method_name, AzureAppConfigurationClientProxy
+        super(AppConfigurationClientTest, self).__init__(method_name)
+        self.vcr.match_on = ["path", "method", "query"]
+        if self.is_playback():
+            connection_str = "Endpoint=https://fake_app_config.azconfig-test.io;Id=0-l4-s0:h5htBaY5Z1LwFz50bIQv;Secret=bgyvBgwsQIw0s8myrqJJI3nLrj81M/kzSgSuP4BBoVg="
+        else:
+            connection_str = os.getenv('APP_CONFIG_CONNECTION')
+        self.app_config_client = AzureAppConfigurationClient.from_connection_string(connection_str)
+
+    def setUp(self):
+        super(AppConfigurationClientTest, self).setUp()
+        self.test_config_setting = self._add_for_test(
+            ConfigurationSetting(
+                key=KEY,
+                label=LABEL,
+                value=TEST_VALUE,
+                content_type=TEST_CONTENT_TYPE,
+                tags={"tag1": "tag1", "tag2": "tag2"},
+            )
+        )
+        self.test_config_setting_no_label = self._add_for_test(
+            ConfigurationSetting(
+                key=KEY,
+                label=None,
+                value=TEST_VALUE,
+                content_type=TEST_CONTENT_TYPE,
+                tags={"tag1": "tag1", "tag2": "tag2"},
+            )
+        )
+        self.to_delete = [self.test_config_setting, self.test_config_setting_no_label]
+
+    async def tearDown(self):
+        super(AppConfigurationClientTest, self).tearDown()
+        for item in self.to_delete:
+            await self.app_config_client.delete_configuration_setting(
+                key=item.key, label=item.label
+            )
+
+    async def _add_for_test(self, kv):
+        exist = bool(
+            list(
+                await self.app_config_client.list_configuration_settings(
+                    keys=[kv.key], labels=[kv.label]
+                )
+            )
         )
+        if exist:
+            self._delete_from_test(kv.key, kv.label)
+        return await self.app_config_client.add_configuration_setting(kv)
+
+    async def _delete_from_test(self, key, label):
+        try:
+            await self.app_config_client.delete_configuration_setting(key=key, label=label)
+        except AzureError:
+            logging.debug(
+                "Error occurred removing configuration setting %s %s during unit test"
+                % (key, label)
+            )
 
     # method: add_configuration_setting
-    def test_add_configuration_setting(self):
+    async def test_add_configuration_setting(self):
```

Comment:
None of your async tests are running:
```
sdk/appconfiguration/azure-appconfiguration/tests/test_azure_configuration_client_aad_async.py::AppConfigurationClientTest::test_delete_read_only
sdk/appconfiguration/azure-appconfiguration/tests/test_azure_configuration_client_async.py::AppConfigurationClientTest::test_delete_read_only
  /opt/hostedtoolcache/Python/3.7.5/x64/lib/python3.7/unittest/case.py:628: RuntimeWarning: coroutine 'AppConfigurationClientTest.test_delete_read_only' was never awaited
    testMethod()
```

See async storage tests for example of decorator to make them run. As long as you have a message like this one, it means they don't run.
Also, tearDown can't be async, so if you need async code in teardown, use a "get_running_loop" to run it.